estela-api/api/exceptions.py

```diff
@@ -1,8 +1,7 @@
+from api import errors
 from rest_framework import status
 from rest_framework.exceptions import APIException
 
-from api import errors
```

Comment:
Let's revert these import sorts in files where this is the only change. The package is not correctly telling apart local from third-party packages, resulting in it grouping them like here. We want our code to be imported last to avoid any problems.

Reply:
Done.